cas-server-support-jpa-ticket-registry/src/main/java/org/apereo/cas/config/JpaTicketRegistryConfiguration.java

```diff
@@ -47,6 +47,7 @@
     public String[] ticketPackagesToScan() {
         return new String[] {
                 "org.apereo.cas.ticket",
+                "org.apereo.cas.util.jpa",
```

Comment:
![image](https://cloud.githubusercontent.com/assets/5005199/18124492/10fe6110-6fa5-11e6-9217-fd97cddf77f6.png)

Without this conversion, the database field type will be oid， but we hope that timestamp